### PR TITLE
fix htmlparser

### PR DESCRIPTION
**Pull request by bung87**
need to set `eol=lf` in `.gitattributes` for testing compare html content from file.

### CODE REVIEW

File: src/fusion/htmlparser/xmltree.nim [hunk 1]
```diff
@@ -633,8 +655,11 @@ proc add*(result: var string, n: XmlNode, indent = 0, indWidth = 2,
         result.addEscapedAttr(val)
         result.add('"')
 
-    if n.len == 0:
-      result.add(" />")
+    if n.len == 0 and n.fTag in SingleTags:
+      result.add("/>")
+      # result.add("</")
+      # result.add(n.fTag)
+      # result.add(">")
```
Comment (narimiran) [inline src/fusion/htmlparser/xmltree.nim:662]: What's with these added commented-out lines?

Comment (bung87) [reply src/fusion/htmlparser/xmltree.nim:662]: it"s single tag element , so it is self closed , no need add end tag.

Comment (narimiran) [reply src/fusion/htmlparser/xmltree.nim:662]: Ok, but my question is: why do we need those commented-out lines here? Should this be cleaned up?

Comment (bung87) [reply src/fusion/htmlparser/xmltree.nim:662]: yeah, should be cleaned up. I'll do it if this PR can be merged.

Comment (narimiran) [reply src/fusion/htmlparser/xmltree.nim:662]: It cannot be merged, as it has merge conflicts (which I cannot resolve in Github's web interface), so can you solve that? Or maybe just take the updated `master` branch and apply your changes?

Comment (bung87) [reply src/fusion/htmlparser/xmltree.nim:662]: okay, I'll do that.

**Comment by bung87** [comment]
close in flavor of https://github.com/nim-lang/fusion/pull/101

**Pull request closed without merging**